src/hooks/useMultiSelect.stories.mdx

```diff
@@ -47,7 +52,7 @@ export default function MultiSelectWithHookExample() {
       variant="listbox"
       items={items}
       selectedItems={selectedItems}
-      onChange={(selectedItem) => onChange(selectedItem.id, multiSelectId)}
+      onChange={(e) => onChange(e.target.id, multiSelectId)}
```

Comment:
This was fine before. For "listbox", the first value is of type `MultiSelectItem`.